.babelrc

```diff
@@ -2,7 +2,7 @@
   "presets": [
     ["env", {
       "targets": {
-        "node": "4.5"
+        "node": 6
```

Comment:
it will transform async function to generator instead, without regeneratorRuntime.